assets/javascripts/discourse/components/chat-selection-manager.js

```diff
@@ -11,17 +13,27 @@ import getURL from "discourse-common/lib/get-url";
 
 export default class AdminCustomizeColorsShowController extends Component {
   tagName = "";
-  @service router;
   chatChannel = null;
   selectedMessageIds = null;
   showChatQuoteSuccess = false;
   cancelSelecting = null;
 
+  @service router;
+  @alias("currentUser.staff") showMoveMessages;
```

Comment:
This can be a confusing use of `alias`, could we use `currentUser.staff` in the template directly?

Reply:
Yes I missed this one, definitely not an alias anyways; but a reads if we keep it. I do agree just `currentUser.staff` is probably more explicit in this case.

Reply:
Okay will change to using the staff directly in the template